sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -195,163 +195,39 @@ apple_iap_subscriptions AS (
     subplat.product_id = "org.mozilla.ios.FirefoxVPN"
     AND subplat.fxa_uid IS NOT NULL
 ),
-android_iap_events AS (
```

Comment:
Seems like a major logic change, would we ever need to run backfills on this table?

Reply:
This is intended to result in no logic change at all, just moving most of this Google subscriptions logic to `subscription_platform_derived.google_subscriptions_v1`.  It isn't a strict lift-and-shift because some logic is still more appropriate to here (e.g. `ended_at`, `pricing_plan`).  I basically followed the example of what @relud did for extracting the bulk of the Apple subscriptions logic out of this ETL in [PR 3329](https://github.com/mozilla/bigquery-etl/pull/3329/files#diff-0fc843607b426568a5e05dce5ecac54e500465668ca9331abf61b4d32255e536).

Reply:
And in terms of backfills, that isn't really a concern here because the entire table is rebuilt every run.